index.js

```diff
@@ -66,7 +66,42 @@ function toggleColor(element) {
         element.style.color = "green"
     }
 }
+//added event listener
+header.addEventListener('click', function(event) {
+    
+   toggleColor(event.target)
+})
 
 /***** Deliverable 2 *****/
+//find button and like content
+let button = document.querySelector('button.like-button')
+
+button.addEventListener('click', function () {
+
+    likes.textContent = `${traveler.likes ++} Likes`
+})
 
 /***** Deliverable 3 *****/
+form = document.querySelector('form#new-animal-sighting-form')
```

Comment:
be careful about using const and let to declare your variables. Using global variables can introduce hard to find bugs.

Reply:
I would also make the variable very specific. `animalSightingForm`